scripts/bash_aliases.sh

```diff
@@ -29,3 +29,8 @@ alias subfmt="python2.7 -m flake8 --ignore E731 --max-line-length=120 --exclude=
 
 # Set robot model param, for using rviz when playing bags
 alias submodel="rosparam set /robot_description -t \$CATKIN_DIR/src/SubjuGator/command/sub8_missions/tools/sub8/sub8.xml"
+
+# Thrusters
+alias thruster_debug_shell="rosrun sub8_diagnostics thruster_debug_shell.py -i"
```

Comment:
The convention for the rest of our aliases are the start them with sub. We should probably follow that for these as other MIL projects have thrusters